Traceback (most recent call last):
  File "/tmp/tmpzl7lj860", line 30, in <module>
    with open('selected_four_letter_words.txt') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'selected_four_letter_words.txt'